README.md

```diff
@@ -26,89 +44,57 @@ We welcome questions/comments/feedback (and pull requests).
-center workloads (containers, VMs, or bare metal).  It is based on the same 
-scalable IP networking principles as the internet, providing connectivity using
-standard IP routing and isolation between workloads (or other fine grained
-policy) using iptables programmed at the source and destination workloads.
-
-Read more about it on the [Project Calico website][project-calico].
-
-Project Calico uses [etcd][etcd] to distribute information about workloads, 
-endpoints (a specific networking interface associated with a workload),
-and policy to each Docker host.
-
-The `calico-node` service is a worker that configures the network endpoints 
-for containers, handles IP routing, and installs policy rules.  It runs in its 
-own Docker container, and comprises
-- Felix, the Calico worker process
-- BIRD, the route distribution process
-
-We provide a command line tool, `calicoctl`, which makes it easy to configure 
-and start the Calico services listed above, and allows you to interact with 
-the etcd datastore to define and apply network and security policy to the 
-containers you create. Using `calicoctl`, you can provision Calico nodes, 
-endpoints, and define and manage a rich set of security policy. 
-
 ## Getting Started
 
-To get started using, we recommend running through one or more of the available 
-[demonstrations](#demonstrations) described below.
-
-If you would like to get involved writing code for calico-docker, or if you 
-need to build binaries specific to your OS, checkout out the 
-[Building and testing guide](docs/Building.md).
-
-### Demonstrations
-
-Worked examples are available for demonstrating Calico networking with the 
-following different networking options:
+To get started using Calico, we recommend running through one or more of the 
+available getting started guides linked below.
 
-- [Demonstration with Docker default networking](docs/getting-started/default-networking/Demonstration.md)
-- [Demonstration with libnetwork](docs/getting-started/libnetwork/Demonstration.md)
+These guides will help you understand the different environment options when 
+using Calico.  In most of these guides we provide details for running the
+demonstrations using manual setup on your own servers, and a quick set-up in a 
+virtualized environment using Vagrant or a number of cloud services.
 
-See the [Networking options](#networking-options) below for more details on 
-each of these different networking options.
-
-With each of these tutorials we provide details for running the demonstration 
-using manual setup on your own servers, or with a quick set-up in a virtualized
-environment using Vagrant, or a number of cloud services.
-
-We also provide the following additional demonstration: 
+- [Calico networking as a Docker network plugin](docs/getting-started/docker-network-plugin/GettingStarted.md)
+- [Calico networking with no Docker networking](docs/getting-started/no-docker-networking/GettingStarted.md)

```

Comment:
Maybe would be nice if this could capture that fact that we're still doing docker, just not using it for networking.

I'm thinking about what this list will look like when we add 
* Calico networking with rkt